app/controllers/v0/profile/connected_applications_controller.rb

```diff
@@ -8,12 +10,29 @@ def index
       end
 
       def destroy
-        app = OktaRedis::App.with_id(connected_accounts_params[:id])
-        app.user = @current_user
+        icn = @current_user.icn
+        client_id = :id
+
+        if icn.nil? || client_id.nil?
+          render json: { error: 'icn and/or client_id is missing' }
+          return
+        end
+
+        root_url = request.base_url == 'https://api.va.gov' ? 'https://api.va.gov' : 'https://sandbox-api.va.gov'
+        revocation_url = "#{root_url}/internal/auth/v3/user/consent"
 
-        app.delete_grants
+        payload = { icn:, client_id: }
```

Comment:
I almost missed this. "client_id" should be "clientId.

Reply:
Now using clientId as the key instead of client_id